pkg/cvo/updatepayload.go

```diff
@@ -264,6 +264,9 @@ func (r *payloadRetriever) fetchUpdatePayloadToDir(ctx context.Context, dir stri
 				setContainerDefaults(corev1.Container{
 					Name:    "rename-to-final-location",
 					Command: []string{"mv", tmpDir, dir},
+					SecurityContext: &corev1.SecurityContext{
+						ReadOnlyRootFilesystem: ptr.To(true),
+					},
```

Comment:
Should the init containers (L236-L262) have this too? If not, can the commit + pr description contain reasoning for that?. 

Or even better, change the `setContainerDefaults` to default to `true` so that the right thing is default and whatever needs a `false` would need to override?